apps/shipments/migrations/0037_set_initial_historical_objects.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.1.7 on 2019-05-08 18:01
+
+from django.db import migrations
+from apps.shipments.models import Shipment, Location
+
+
+def set_initial_historical(apps, schema_editor):
+    """
+    Create an initial  HistoricalShipment, HistoricalLocation
+    object for every shipment, location respectively with a null history_user
+    """
+
```

Comment:
Consider using the methods provided by the library to populate the initial history: https://github.com/treyhunner/django-simple-history/blob/master/simple_history/management/commands/populate_history.py

Reply:
I cannot use this module, since we are setting the initial `author=None`.